vpr/src/place/analytical_placement/AnalyticalSolver.h

```diff
@@ -26,6 +28,20 @@ class QPHybridSolver : public AnalyticalSolver {
 // need to store the matrices for the system of equations in the class.
 // TODO: Store a standard VTR matrix and pass that into Eigen using Map
 public:
-    void solve(PartialPlacement &p_placement) final;
-};
+    QPHybridSolver(PartialPlacement &p_placement);
+    void solve(unsigned iteration) final;
+    PartialPlacement &p_placement;
+    // Constructor fills the following with no legalization
+    Eigen::SparseMatrix<double> A_sparse;
+    Eigen::VectorXd b_x;
+    Eigen::VectorXd b_y;
+    // Difference due to psudo anchor points from legalization
+    Eigen::SparseMatrix<double> A_sparse_diff;
+    Eigen::VectorXd b_x_diff;
+    Eigen::VectorXd b_y_diff;
```

Comment:
I do not think that these three objects need to be stored in the class. They are created and "destroyed" in the same method.